worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+    display_name = "Progressive Water Training"
+    option_none = 0
+    option_basic = 1
+    option_advanced = 2
+
+class ProgressiveEggAim(Choice):
+    """Set to Basic: Third Person Egg Shooting to Egg Aim. 
+    Set to Advanced: Third Person Egg Shooting to Amaze-O-Gaze to Egg Aim to Breegull Blaster.
+    Randomize Moves and Randomize BK Moves are required."""
+    display_name = "Progressive Egg Aim"
+    option_none = 0
+    option_basic = 1
+    option_advanced = 2
+
+class ProgressiveFlight(Toggle):
+    """Flight Pad to Beak Bomb to Airborne Egg Aim. Randomize Moves and Randomize BK Moves are required."""
+    display_name = "Progressive Flight"
+
+class ProgressiveBashAttack(Toggle):
+    """Ground Rat-a-tat Rap to Breegull Bash. Randomize Stop N Swap and Randomize BK Moves are required"""
+    display_name = "Progressive Bash Attack"
+
+class EnableCheatoRewards(DefaultOnToggle):
+    """Cheato rewards you with a cheat and an additional randomized reward. Use Cheato Pages as Filler cannot be set if this is enabled."""
+    display_name = "Cheato Rewards"
+
+class EnableMultiWorldJinjos(DefaultOnToggle):
+    """Jinjos have fled to other worlds. Other players need to return them home."""
+    display_name = "Randomize Jinjos"
+
+class EnableMultiWorldDoubloons(Toggle):
+    """Jolly Roger's Doubloons are scattered across the MultiWorld."""
+    display_name = "Randomize Doubloons"
+
+class EnableMultiWorldCheatoPages(DefaultOnToggle):
+    """Cheato pages are scattered across the MultiWorld."""
+    display_name = "Randomize Cheato Pages"
+
+class EnableMultiWorldHoneycombs(DefaultOnToggle):
+    """Honeycombs are scattered across the MultiWorld."""
+    display_name = "Randomize Honeycombs"
+
+class EnableHoneyBRewards(DefaultOnToggle):
+    """Honey B gives you health and an additional randomized reward."""
+    display_name = "Honey B Rewards"
+
+class EnableMultiWorldGlowbos(DefaultOnToggle):
+    """Glowbos are scattered across the MultiWorld."""
+    display_name = "Randomize Glowbos"
+
+class EnableMultiWorldTrebleClefs(DefaultOnToggle):
+    """Treble Clefs are scattered across the MultiWorld."""
+    display_name = "Randomize Treble Clefs"
+
+class EnableMultiWorldTrainStationSwitches(Toggle):
+    """Train Stations are scattered across the MultiWorld."""
+    display_name = "Randomize Train Station Switches"
+
+class EnableMultiWorldChuffyTrain(Toggle):
+    """Chuffy is lost somewhere in the MultiWorld. 
+    Once received, you can call Chuffy at any unlocked station without defeating Old King Coal."""
+    display_name = "Chuffy as a randomized AP Item"
+
+class EnableMultiWorldNotes(Toggle):
+    """Note Nests are scattered across the MultiWorld."""
+    display_name = "Randomize Note Nests"
+
+class BassclefNotes(Range):
+    """Convert some 5 notes into Bassclefs (10 notes). How many notes do you want converted?
+       Be aware that 1 bassclef removes two 5 notes and adds an additional Big-O-Pants.
```

Comment:
```suggestion
       Be aware that 1 Bassclef removes two 5 notes and adds an additional Big-O-Pants.
```
Bassclef is capitalized in the previous sentence, so it should be capitalized here too

Reply:
Bass Clef has been capitalised and split into 2 words everywhere except the connector. I'll let jj take care of the connector.